rts/motoko-rts/src/ic0_stable.rs

```diff
@@ -0,0 +1,55 @@
+extern "C" {
```

Comment:
Just some structuring idea, considering that `ic0_stable` is a helper for `region0` and `region`: Maybe, the entire region support could be more hierarchically organized as one module, with sub-modules, covering logic from `ic0_stable`, `region0` and `region`.